ksqldb-engine/src/main/java/io/confluent/ksql/engine/PullQueryExecutionUtil.java

```diff
@@ -99,12 +99,19 @@ private static SourceName getSourceName(final ImmutableAnalysis analysis) {
   }
 
   private static KsqlException notMaterializedException(final SourceName sourceTable) {
+    final String tableName = sourceTable.toString().replaceAll("`", "");
     return new KsqlException(
-        "Can't pull from " + sourceTable + " as it's not a materialized table."
-            + PullQueryValidator.PULL_QUERY_SYNTAX_HELP
+            "The " + sourceTable + " table isn't queriable. To derive a queriable table, "
+                    + "you can do 'CREATE TABLE QUERIABLE_"
+                    + tableName
+                    + " AS SELECT * FROM "
+                    + tableName
+                    + "."
```

Comment:
```suggestion
                    + "'."
```

(my entry for most cryptic feedback of the year)